apis/hive/v1/zz_generated.deepcopy.go

```diff
@@ -2402,6 +2403,11 @@ func (in *HiveConfigSpec) DeepCopyInto(out *HiveConfigSpec) {
 		*out = new(FeatureGateSelection)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.MetricsConfig != nil {
+		in, out := &in.MetricsConfig, &out.MetricsConfig
+		*out = new(metricsConfig.MetricsConfig)
+		(*in).DeepCopyInto(*out)
```

Comment:
Cannot for the life of me figure out what small change am I missing to generate `v1/metricsConfig/zz_generated.deepcopy.go` automatically

Reply:
codegen for submodules lives in hack/update-codegen.sh. This should do the trick:
```diff
diff --git a/hack/update-codegen.sh b/hack/update-codegen.sh
index d0da2f89c..e829afffd 100755
--- a/hack/update-codegen.sh
+++ b/hack/update-codegen.sh
@@ -19,7 +19,7 @@ GOFLAGS="" bash ${CODEGEN_PKG}/generate-groups.sh "all" \
 GOFLAGS="" bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy" \
   github.com/openshift/hive/pkg/client \
   github.com/openshift/hive/apis \
-  "hive:v1/agent hive:v1/aws hive:v1/azure hive:v1/baremetal hive:v1/gcp hive:v1/none hive:v1/openstack hive:v1/ovirt hive:v1/vsphere hive:v1/ibmcloud hivecontracts:v1alpha1" \
+  "hive:v1/agent hive:v1/aws hive:v1/azure hive:v1/baremetal hive:v1/gcp hive:v1/metricsConfig hive:v1/none hive:v1/openstack hive:v1/ovirt hive:v1/vsphere hive:v1/ibmcloud hivecontracts:v1alpha1" \
   --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt \
   --trim-path-prefix github.com/openshift/hive \
   ${verify}
```